### PR TITLE
Revert 4 read me raw data update

### PR DESCRIPTION
**Pull request by morganjacobus**

### CODE REVIEW

**Comment by morganjacobus** [comment]
I think this can be closed. All the changes in here are contained within the clean_data_script branch, which can be merged into the main one the pull request is made.

File: .Rprofile [hunk 1]
```diff
@@ -13,69 +16,60 @@
...
-  calendar <- calendar %>% mutate(.,service_type = 
-                                    with(.,case_when((service_id == 3) ~ "Saturday",
-                                                     (service_id == 4) ~ "Sunday",
-                                                     (service_id == 5) ~ "Weekday")))
-  # Convert route_type to bus or rail
-  routes <- routes %>% mutate(.,route_type_desc = 
-                                with(.,case_when((route_type == 3) ~ "Bus",
-                                                 (route_type == 1) ~ "Rail")))
+  ## SUMMARIZE STOP ROUTE DATA ----------------------------------------------------------------------------------------
+  route_data_2019 <- gtfs_dec_2019_df_list[[7]]
```
Comment (QYAN24) [inline .Rprofile:28]: I think we need an explanation in a comment why we need the two sets of GTFS data.

File: code/functions/bus_stop_census_code_functions.R [hunk 2]
```diff
@@ -0,0 +1,104 @@
+
+  correctRecords <- function(survey_df, corrections_df, source_sheet) {
+    # The correctRecords functions applies corrections specified in the Bus Stop Census Corrections Log, located here,
+    # https://docs.google.com/spreadsheets/d/1n8RErNhOXrNptVyul5rTuE-ZTdSzkljB0cCAV8hBJkw/edit?usp=sharing,
+    # to the raw data.
+    corrections_subset <- subset(corrections_df, Correction_Source_Sheet == source_sheet) # Subset corrections data based on the sheet the corrections originated from
```
Comment (QYAN24) [inline code/functions/bus_stop_census_code_functions.R:6]: Move the comment to above the statement.

File: code/functions/bus_stop_census_code_functions.R [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    calendar <- calendar %>% mutate(.,service_type = 
+                                                        with(.,case_when((service_id == 3) ~ "Saturday",
+                                                                         (service_id == 4) ~ "Sunday",
+                                                                         (service_id == 5) ~ "Weekday")))
+    # Convert route_type to bus or rail
+    routes <- routes %>% mutate(.,route_type_desc = 
+                                                    with(.,case_when((route_type == 3) ~ "Bus",
+                                                                     (route_type == 1) ~ "Rail")))
+    
+    # Format Stop Times To be 24 hour
```
Comment (QYAN24) [inline code/functions/bus_stop_census_code_functions.R:85]: This comments should probably be something like "Convert stop hours that are into the next day to regular clock hours (e.g. 25:12:05 becomes 01:12:05)"

File: code/raw_code/01_clean_data.R [hunk 4]
```diff
@@ -1,51 +1,266 @@
...
+  
+  # Convert Timestamp To R Readable Format | Data Cleaning Task #7
+  raw_df_clean_deduped$Timestamp_Sub <- substring(raw_df_clean_deduped$Timestamp, 0, 24) # Take substring of timestamp to reformat
+  raw_df_clean_deduped$Timestamp_Sub <- as.POSIXct(raw_df_clean_deduped$Timestamp_Sub, format = "%a %b %d %Y %H:%M:%S") # Convert to POSIXct
+  raw_df_clean_deduped <- raw_df_clean_deduped %>% mutate(Timestamp = Timestamp_Sub) %>% # Set original timestamp value to converted timestamp
+                                                   select(-Timestamp_Sub) # Remove timestamp helper field created for the conversion
+  
+
+  ## INTEGRATE SURVEYOR SUBMITTED CORRECTIONS -----------------------------------------------------------------------------------
+  # Use correctRecords function to apply corrections to respose data | Data Cleaning Task #8
```
Comment (QYAN24) [inline code/raw_code/01_clean_data.R:80]: Typo 'response'

**Review by QYAN24 — APPROVED** [review]
Some minor cosmetic suggestions. This looks well written.

**Comment by morganjacobus** [comment]
Hi @QYAN24 ,
Thank you for the suggestions. I will incorporate them into the clean_data_script branch. This was an a branch I created on accident. I plan on adding more explanation about the GTFS in the code and in the read me.

**Pull request merged**